app/services/youtube_service/client.rb

```diff
@@ -0,0 +1,38 @@
+class YoutubeService
+  class Client
+    API_PREFIX = 'https://www.googleapis.com/youtube/v3'.freeze
+    CHANNEL_URL = Addressable::Template.new(
+      "#{API_PREFIX}/channels.list{?query*}"
+    ).freeze
```

Comment:
Url here seems to be `https://www.googleapis.com/youtube/v3/channels` (Ref: https://developers.google.com/youtube/v3/docs/channels/list)

```json
{
    "kind": "youtube#channelListResponse",
    "pageInfo": {
        "totalResults": 1,
        "resultsPerPage": 1
    },
    "items": [{
        "kind": "youtube#channel",
        "id": "UCcacpXYQrVx6ywO0QxW5nNQ"
    }]
}
```